stdlib/src/builtin/simd.mojo

```diff
@@ -576,13 +580,47 @@ struct SIMD[type: DType, size: Int = simdwidthof[type]()](
 
         return String.format_sequence(self)
 
+    @always_inline
+    fn __repr__(self) -> String:
+        """Get the representation of the SIMD value eg. "SIMD[DType.int8, 2](1, 2)".
+        Returns:
+            The representation of the SIMD value.
+
+        """
+        var values: String
```

Comment:
Could this `values` be replaced with a `StringRef` or `StringSlice`? Or does the concatenation not work then?

Reply:
Wouldn't that be unsafe/unnecessary since assigning `values` is the final use of `output` and it's being moved anyway?

Reply:
The `size > 1` branch will probably need to make a copy. `StringSlice` should be safe btw.